freezeyt_blog/app.py

```diff
@@ -13,7 +13,7 @@
 @app.route('/')
 def index():
     """Start page with list of articles."""
-    post_names = [art_file.stem for art_file in ARTICLES_PATH.iterdir()]
+    post_names = [a.stem for a in ARTICLES_PATH.iterdir() if a.is_file()]
```

Comment:
I think it would be even better to check if the [suffix](https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.suffix) is `.md`.
Also, [`iterdir`](https://docs.python.org/3/library/pathlib.html#pathlib.Path.iterdir) gives the files in arbitrary order, which can can be different depending on the OS. Consider [`sorted()`](https://docs.python.org/3/library/functions.html#sorted), as in `sorted(a.stem for a in ... if a.is_file())`.

Reply:
thanks, I take look at it

Reply:
I added variant if we will decide in future for more categories, where every single category will represent subdir in dir `articles`.